crates/rpc/rpc/src/eth/filter.rs

```diff
@@ -14,6 +20,9 @@ use std::{collections::HashMap, sync::Arc, time::Instant};
 use tokio::sync::Mutex;
 use tracing::trace;
 
+/// The default maximum of logs in a single response.
+const DEFAULT_MAX_LOGS_IN_RESPONSE: usize = 1_000;
```

Comment:
@prestwich as we're getting into log filtering territory, would love to get your thoughts on what ideal behavior should be for these

Reply:
user code commonly has cases at 10,000 (infura on eth mainnet) and 2,000 (everyone else)